sw/host/tests/mask_rom/e2e_bootup_bad_rom_ext_signature/src/main.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, Result};
+use regex::Regex;
+use std::io::Write;
+use std::path::PathBuf;
+use std::time::Duration;
+use structopt::StructOpt;
+use tempfile::NamedTempFile;
+
+mod helpers;
+use helpers::init::InitializeTest;
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::uart::console::{ExitStatus, UartConsole};
+
+#[derive(Debug, StructOpt)]
+struct Opts {
+    #[structopt(flatten)]
+    init: InitializeTest,
+
+    #[structopt(
+        long, parse(try_from_str=humantime::parse_duration),
+        default_value = "60s",
+        help = "Duration of ROM detection timeout",
```

Comment:
I think this help text is wrong.  This is really the rom_ext boot detection timeout, correct?
I also think this timeout is probably way too long, at least for cw310.  I think this is probably more like 5s. (In reality the rom_ext boots in less than a second)

Reply:
I've updated the help text to `ROM_EXT boot detection timeout` and changed the default to 5s.